roles/storage-cinder/defaults/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+action: provision
+namespace: openstack
+service_account: cinder
+privileged_service_account: cinder-privileged
+
+storage_cinder_template_dir: "{{ role_path }}/templates"
+
+mariadb_root_user: root
+mariadb_root_password: weakpassword
+
+cinder_user: cinder
+cinder_password: cinderpassword
+
+rabbimq_user: guest
+rabbitmq_password: rabbitmqpassword
+
+auth_strategy: noauth
+
+cinder_config:
+  DEFAULT:
+    public_bind_host: "0.0.0.0"
+
+role_name: cinder
+
+infra_node_label: mariadb
```

Comment:
let's make this label less likely to collide:  perhaps kubevirt.io/storage-cinder-mariadb-host